client/src/shared-components/navbar.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import Media from 'react-media';
+
+import DesktopNav from './desktop-nav';
+import Header from './header';
+import MobileNav from './mobile-nav';
+
+// width of iPad in portrait
+const minWidth = 769;
```

Comment:
If this line happens more frequently, you should put things like this in a constants file under utils